front-end-web/src/Components/Pages/AuthorizationPage.tsx

```diff
@@ -9,11 +11,24 @@ const AuthorizationPage: React.FunctionComponent<Props> = ({
   handlers,
   classes
 }) => {
+  const useStyles = makeStyles(theme => ({
+    button: {
+      margin: "8px"
+    },
+    input: {
+      display: "none"
+    }
+  }));
+
+  const classes2 = useStyles();
   return (
     <Fragment>
-      {
-        //TODO
-      }
+      <Typography>
+        Press this button to log in to the Authorization page
+      </Typography>
+      <Button variant="outlined" color="primary" className={classes.button}>
+        Primary
```

Comment:
Would you be able to switch the button text from "Primary" to something like "Login and Authorize"?